modules/context/api.go

```diff
@@ -89,25 +93,19 @@ func (ctx *APIContext) ServerError(title string, err error) {
 // Error responds with an error message to client with given obj as the message.
 // If status is 500, also it prints error to log.
 func (ctx *APIContext) Error(status int, title string, obj interface{}) {
-	var message string
+	var foundError error
+	message := title
 	if err, ok := obj.(error); ok {
-		message = err.Error()
+		foundError = err
 	} else {
 		message = fmt.Sprintf("%s", obj)
 	}
 
 	if status == http.StatusInternalServerError {
-		log.ErrorWithSkip(1, "%s: %s", title, message)
-
-		if setting.IsProd && !(ctx.Doer != nil && ctx.Doer.IsAdmin) {
-			message = ""
-		}
+		log.ErrorWithSkip(1, "%s: %w", title, foundError)
```

Comment:
%w doesn't work in log.

```suggestion
		log.ErrorWithSkip(1, "%s: %v", title, foundError)
```